docs/development/shareable-configs.md

```diff
@@ -54,44 +54,40 @@ You can set a Git tag (like a SemVer) to use a specific release of your shared c
 
 #### GitHub
 
-| name                                        | example use                      | preset    | resolves as                  | filename        | Git tag        |
```

Comment:
Wrong merge?

Reply:
I tried to go back to a previous commit

Reply:
still something wrong, you should rebase onto main and force push .